### PR TITLE
chore(deps): update to iroh v0.7.0

### PR DESCRIPTION
**Pull request by b5**

### CODE REVIEW

**Comment by dignifiedquire** [comment]
please regenerate the bindings, trying to keep the comitted things up to date

**Comment by b5** [comment]
I'm glad you asked. found a few broken APIs

**Comment by dignifiedquire** [comment]
you are using an outdated generator for go

File: src/node.rs [hunk 1]
```diff
@@ -517,7 +522,7 @@ impl Doc {
 
     pub fn stop_sync(&self) -> Result<(), Error> {
         block_on(&self.rt, async {
-            self.inner.stop_sync().await.map_err(Error::doc)?;
+            self.inner.leave().await.map_err(Error::doc)?;
```
[the remark this answers is not in the document]
Comment (dignifiedquire) [reply src/node.rs:525]: @Frando please confirm, I believe yes

**Comment by b5** [comment]
PTAL, pretty sure I'm now using the latest for go. built from `2f696957a1ea2284e580109d3fd95b1d39aea141` of your fork

**Review by dignifiedquire — APPROVED** [review]

**Pull request merged**